src/service-worker.js

```diff
@@ -43,7 +43,7 @@ registerRoute(
 
     return true;
   },
-  createHandlerBoundToURL(process.env.PUBLIC_URL + '/index.html')
+  createHandlerBoundToURL(process.env.VITE_PUBLIC_URL + '/index.html')
```

Comment:
Should this use the `import.meta.env` convention like the other env variables? I've never used Vite so I genuinely don't know😅

Reply:
Ahh yep that needs changing